client/my-sites/checkout/src/lib/pending-page.ts

```diff
@@ -23,6 +24,7 @@ export interface RedirectForTransactionStatusArgs {
 	redirectTo?: string;
 	siteSlug?: string;
 	saasRedirectUrl?: string;
+	fromSite?: string;
```

Comment:
What's the difference between `fromSite` and `fromSiteSlug`? 🤔

Reply:
> What's the difference between `fromSite` and `fromSiteSlug`? 🤔

`fromSiteSlug` and `fromSite` are both the same value- the site's slug(ultimately originating from the 'not-connected yet' plugin). The only difference is that `fromSiteSlug` comes from a url query arg on route `/checkout/jetpack/:productSlug` which then get's passed along as a url query arg `fromSite` on route `/checkout/thank-you/no-site/pending/:orderId`.
I think I named them slightly different so as to distinguish that they are defined from url query args from different routes.., but maybe this naming difference is not necessary and actually more confusing.
I'll rename `fromSite` to `fromSiteSlug` keeping the name consistent. 👍